app/views/hq/sponsors/index.html.haml

```diff
@@ -0,0 +1,37 @@
+- if @sponsors.empty?
```

Comment:
@PurityControl Agree with your reasoning, but I wouldn't hold up the PR for this.